clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -77,7 +77,7 @@
 
   <xs:simpleType name="connectionUrl">
     <xs:restriction base="xs:anyURI">
-      <xs:pattern value="\w+://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
+      <xs:pattern value="\w+://([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?(,([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?)*(/[^\?#]*)?(\?[^#]*)?(#.*)?"/>
```

Comment:
Pattern looks right.  Not sure I agree with @chrisdennis that a simpler expression might be available ... you're kinda stuck since the replicated part -- `(,([^\]\[/?#@]+@)?[^:?#/]+(:[1-9][0-9]{0,4})?)*` -- begins with a comma so the pattern following it needs replication.